Configure.cmake

```diff
@@ -78,49 +98,46 @@ CHECK_C_SOURCE_COMPILES("
 
 set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_FMA4})
 CHECK_C_SOURCE_COMPILES("
-#if defined(_MSC_VER)
-#include <intrin.h>
-#else
-#include <x86intrin.h>
-#endif
-int main() {
-  __m256d r = _mm256_macc_pd(_mm256_set1_pd(1), _mm256_set1_pd(2), _mm256_set1_pd(3));
-}" COMPILER_SUPPORTS_FMA4)
+  #if defined(_MSC_VER)
+  #include <intrin.h>
+  #else
+  #include <x86intrin.h>
+  #endif
+  int main() {
+    __m256d r = _mm256_macc_pd(_mm256_set1_pd(1), _mm256_set1_pd(2), _mm256_set1_pd(3));
+  }" COMPILER_SUPPORTS_FMA4)
 
 set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_AVX2})
 CHECK_C_SOURCE_COMPILES("
-#if defined(_MSC_VER)
-#include <intrin.h>
-#else
-#include <x86intrin.h>
-#endif
-int main() {
-  __m256i r = _mm256_abs_epi32(_mm256_set1_epi32(1));
-}" COMPILER_SUPPORTS_AVX2)
+  #if defined(_MSC_VER)
+  #include <intrin.h>
+  #else
+  #include <x86intrin.h>
+  #endif
+  int main() {
+    __m256i r = _mm256_abs_epi32(_mm256_set1_epi32(1));
+  }" COMPILER_SUPPORTS_AVX2)
 
 set (CMAKE_REQUIRED_FLAGS ${FLAGS_ENABLE_AVX512F})
 CHECK_C_SOURCE_COMPILES("
-#if defined(_MSC_VER)
-#include <intrin.h>
-#else
-#include <x86intrin.h>
-#endif
-void f(void *p) {
-  _mm512_loadu_si512(p);
-}
-int main() {
-  __m512i a = _mm512_set1_epi32(1);
-  __m512i r = _mm512_andnot_si512(a, a);
-}" COMPILER_SUPPORTS_AVX512F)
+  #if defined(_MSC_VER)
```

Comment:
Please revert the C code. The refactoring does not include cosmetic changes in the code.